packages/wrangler/package.json

```diff
@@ -64,7 +64,7 @@
 		"test": "dotenv -- pnpm run assert-git-version && dotenv -- vitest",
 		"test:ci": "pnpm run test run",
 		"test:debug": "pnpm run test --silent=false --verbose=true",
-		"test:e2e": "dotenv -- vitest -c ./e2e/vitest.config.mts",
+		"test:e2e": "dotenv -- vitest run -c ./e2e/vitest.config.mts",
```

Comment:
Just a quality of life improvement for local development. Without this the test:e2e goes into interactive mode and never exists.